subsys/spm/Kconfig

```diff
@@ -9,6 +9,7 @@ menu "SPM"
 config SPM
 	bool "Use Secure Partition Manager"
 	default y if TRUSTED_EXECUTION_NONSECURE
+	select FW_METADATA
```

Comment:
IMO this should be under the new service config instead of here.

Reply:
Then it will not enable FW_METADATA for the user (ie the root application) only for SPM

Reply:
Ah, I get it. It shouldn't be necessary to enable it just to use the h file, but we can fix that later with `configure_file` or something.

Reply:
I agree